wiki/en/en-Directory-Servers.md

```diff
@@ -9,14 +9,14 @@ permalink: "/wiki/Directory-Servers"
 
 # Public Server Registration
 
-Because there is a limit of 150 servers per directory server, you need to select which directory server you want to advertise your server on. Select a genre if you intend to limit players to that style (players can then see your server in their genre list). 
+Because there is a limit of 150 servers per directory server, you need to select which directory server you want to advertise your server on. Select a genre if you intend to limit players to that style (players can then see your server in their genre list).
 
 When you start a public server, you will see a message saying whether your server has been successfully listed or not. If not, and you leave your server running, it will keep trying to register until a free slot becomes available.  
 
 
 ## Headless servers
 
-Set central (genre) server using the `--centralserver` (`-e`) option as follows:
+Set directory (genre) server using the `--directoryserver` (`-e`) option as follows:
```

Comment:
Yes.

Reply:
so there are other places to change. in headless ? or somewhere like that.

Reply:
Probably yes. The old `--centralserver` argument still works but is deprecated.

Reply:
But it is the one in command-line.md

Reply:
Yes, this was one place where I found the old argument was still used. Maybe @gilgongo knows more here?

Reply:
Sorry, not sure I understand. I replaced all references to "central" servers to "directory" in the docs for the next release, but I think @pljones do the same for the code base. I seem to recall he deprecated it though.

Reply:
Yes. Probably this one got missed somehow

Reply:
It is not only this one.
I have updated all the missing ones in FR language but there is a lot more in en, de, es for the command lines in several files.
Do you want me to list that for you ?
Using grep command make the job of listing.

Reply:
Yes, if you want, you can open a pull request to the translate branch.

Reply:
I've found `--centralserver` in several more places. I'll be making a PR today or tomorrow with those changed, the updated ES docs, some anchors fixed in FR and the `category: "hidden"` removed from en-Getting-Started.md (that shouldn't be there, should it?).

Reply:
No. Category hidden should be there. You should keep in touch with @jujudusud to ensure that you don’t both do the same work.

Reply:
> No. Category hidden should be there

Ok.

I now see @jujudusud posted right before me about the same issue. If you're already on to it, go ahead. Don't worry about ES because I've already done those.

Reply:
>  Category hidden should be there.

Bear in mind that once all the translations are in, I'm re-naming Setup to Getting Started. Right now Getting Started is a redirect to Setup (which is a mistake), hence it's hidden.

Reply:
Also - ugh - I just realised of course that the reason the CLUI option `--centralserver` was still in the changes branch is that I didn't want to change it until it had changed in the code.

Reply:
> Bear in mind that once all the translations are in, I'm re-naming Setup to Getting Started. Right now Getting Started is a redirect to Setup (which is a mistake), hence it's hidden.

But... Setup doesn't exist any more on the translate branch. That's why I assumed Getting started didn't have to be hidden.

OMG, I'm really looking forward to getting past this release...

Reply:
> Setup doesn't exist any more on the translate branch

I already renamed it with this PR and checked for links to setup

Reply:
As far as I am concerned, I have finished the translation of the FR part of the website. I am at the same level as this PR + the modifications of the last PR #493

Reply:
Ok, then `category: "hidden"` has to be added back into FR and ES because they don't have it.

Reply:
No. Category hidden should be removed from all Getting-Started pages.